pkg/acl/access_control_list_middleware_test.go

```diff
@@ -170,5 +171,7 @@ func TestAccessControlListMiddleware_Authorize(t *testing.T) {
 func NextHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK) //nolint
 	_, err := io.WriteString(w, "OK")
-	Expect(err).NotTo(HaveOccurred())
```

Comment:
I think dealing with the error and return an http error would be a suitable approach.
If there is an error while writing the string then return a 5XX error with an indicative message

Reply:
Hey @miguelsorianod 
Returning an error is resulting in issues [here](https://github.com/VanillaSpoon/kas-fleet-manager/blob/main/pkg/acl/access_control_list_middleware_test.go#L138)

`cannot convert NextHandler (value of type func(w http.ResponseWriter, r *http.Request) http.Handler) to http.HandlerFunc`

Is this something I could clean up?

~Other alternatives would be:~
```
// NextHandler is a dummy handler that returns OK when QuotaList middleware has passed
func NextHandler(w http.ResponseWriter, r *http.Request) {
	w.WriteHeader(http.StatusOK) //nolint
	_, err := io.WriteString(w, "OK")
	if err != nil {
		http.Error(w, "{\"error\":\"error handling quotalist middleware\"}", http.StatusInternalServerError)
	}
}
```

But this wouldn't stop the tests, and would be overlooked

Reply:
My suggestion was returning it as an http error, not as a Go error. This is, setting the status of the response to 5XX

Reply:
Ah I see, sorry for the misunderstanding, and thanks for clarifying :)